cubedash/summary/_extents.py

```diff
@@ -305,14 +306,22 @@ def refresh_product(
         change_count += engine.execute(
             DATASET_SPATIAL.delete().where(DATASET_SPATIAL.c.id.in_(datasets_to_delete))
         ).rowcount
+
         log.debug(
             "extent_removal.end",
             deleted_count=change_count,
         )
+    if cleanup_invalid_geometry:
+        change_count += engine.execute(
+            DATASET_SPATIAL.delete().where(
+                func.ST_IsValid(DATASET_SPATIAL.c.footprint) is not True
```

Comment:
I think this needs to be `!= true`, it's SQL and not Python.
```suggestion
                func.ST_IsValid(DATASET_SPATIAL.c.footprint) != True
```

Reply:
Use sqlalchemy construct instead : https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.false